<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>数据期</string>
    </property>
    <widget class="QCheckBox" name="phaseAtery">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>动脉期</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="phaseHepaticPortal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>静脉期</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="phaseEquil">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>平衡期</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="phaseNonEnhance">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>平扫期</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>图片质量</string>
    </property>
    <widget class="QCheckBox" name="qualityPoor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>很差</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qualityNormal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>一般</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qualityGood">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>很好</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>肿瘤情况</string>
    </property>
    <widget class="QCheckBox" name="tumorNo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>无肿瘤</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="tumorMinor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>肿瘤少或者小</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="tumorCritical">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>肿瘤多或者大</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>分割是否有误</string>
    </property>
    <widget class="QCheckBox" name="segError">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>分割有误</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="segOK">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>87</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>分割无误</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>自定义滤镜</string>
    </property>
    <widget class="QTextEdit" name="userdefFilter">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>57</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>输入自定义tag</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>层厚</string>
    </property>
    <widget class="QCheckBox" name="thickLess100">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>层厚&lt;100</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="thick100to200">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>层厚100-200</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="thick200to400">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>层厚200-400</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="thickMore400">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>层厚400以上</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>151</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>操作列表</string>
    </property>
    <widget class="QPushButton" name="loadDBButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>114</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>载入数据库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="savePathButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>114</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>输出路径</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="outputNameEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>581</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>数据库查询结果</string>
    </property>
    <widget class="QTableWidget" name="databaseWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>561</width>
       <height>541</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>层数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>描述</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
